### PR TITLE
remove extra css

### PR DESCRIPTION
**Pull request by shalinee-walkover**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/app/reports/components/columnar-report-component/columnar.report.component.html [hunk 1]
```diff
@@ -11,26 +11,26 @@
...
-            <div class="col-md-2 col-sm-6 mb-2 cb">
+            <div class="col-md-2 col-sm-6 mb-2 clear-both">
                 <input type="checkbox" name="showHideOpeningClosingBalance" id="showHideOpeningClosingBalance" [(ngModel)]="exportRequest.showHideOpeningClosingBalance" [value]="true">&nbsp;<label for="showHideOpeningClosingBalance">Opening/Closing Balance</label>
             </div>
-            <div class="col-md-2 col-sm-6 mb-2 cb">
+            <div class="col-md-2 col-sm-6 mb-2 clear-both">
                 <input type="checkbox" name="balanceTypeAsSign" id="balanceTypeAsSign" [(ngModel)]="exportRequest.balanceTypeAsSign" [value]="true">&nbsp;<label for="balanceTypeAsSign">Cr/Dr with +/-</label>
             </div>
-            <div class="col-md-2 col-sm-6 mb-2 cb">
-                <button class="btn btn-pink w127" [disabled]="!isFormValid() || isLoading" (click)="exportReport();">Export</button>
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/reports/components/columnar-report-component/columnar.report.component.html:33]: why this class is removed?

Comment (shalinee-walkover) [reply apps/web-giddh/src/app/reports/components/columnar-report-component/columnar.report.component.html:33]: i think no need to add width, we follow default width

Comment (ravinderwalkover) [reply apps/web-giddh/src/app/reports/components/columnar-report-component/columnar.report.component.html:33]: Meghna asked me to increase width of button that's why i added that class. You should confirm the use before removing.

Comment (shalinee-walkover) [reply apps/web-giddh/src/app/reports/components/columnar-report-component/columnar.report.component.html:33]: meghna confirmed to remove class

**Pull request merged**